setup.py

```diff
@@ -102,19 +102,31 @@ def collect_external_package_data(external_package_data):
     include_dirs.append(f"{EIGEN}")
 
 else:
-    # Opencv
-    opencv_include_dirs = [
-        "/usr/local/opt/opencv/include",  # old opencv brew (v3)
-        "/usr/local/opt/opencv@3/include",  # new opencv@3 brew
-        "/usr/local/include/opencv4",  # new opencv brew (v4)
-        "/usr/include/opencv4",  # opencv (v4) on ubuntu 20.04
-    ]
-    opencv_library_dirs = [
-        "/usr/local/opt/opencv/lib",  # old opencv brew (v3)
-        "/usr/local/opt/opencv@3/lib",  # new opencv@3 brew
-        "/usr/local/lib",  # new opencv brew (v4)
-        "/usr/lib",  # opencv (v4) on ubuntu 20.04
-    ]
+    # Check for anaconda / miniconda install
+    is_conda = 'CONDA_PREFIX' in os.environ
+    if is_conda:
+        print("Using opencv / eigen3 headers from anaconda")
+        conda_path = os.environ['CONDA_PREFIX']
+        opencv_include_dirs = [os.path.join(conda_path, 'include/opencv4')]
+        if not os.path.exists(opencv_include_dirs[0]):
+            print(("Missing opencv headers - please install opencv in this conda\n"
+                   "environment to enable compilation of pupil_detectors binaries\n"
+                   "call: `conda install -c conda-forge opencv`\n"))
```

Comment:
Have you tested this behavior? Maybe it would make sense to raise an error here as the missing directory will lead to errors down the line anyway. Alternatively, one could [post a warning](https://docs.python.org/3/library/exceptions.html#UserWarning), allowing the code to check for missing eigen3 dependencies, too. I do not know how pip handles those warnings, though.

Reply:
I didn't raise an error because the compilation code below will fail with an error when it can't find the library - but I suppose an error at that stage would be clearer. Can do.